lib/PropTransformer.js

```diff
@@ -21,11 +21,15 @@ export default class PropTransformer extends Component {
     );
 
     if (Array.isArray(this.props.children)) {
-      return transformedChildren;
+      return <div>{transformedChildren}</div>;
```

Comment:
no need of this, react 16 supports returning arrays and null

Reply:
i added react@16.0.0 as peer dependency

Reply:
yes, but often times we ignore the install peer dependancy error (like i did) and this will just make it fail safe

Reply:
@abhi-govula can u make it a span, block element affects layout if child was an inline element